api/v1/app.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+"""
+Flask web application api
+"""
+
+from flask import Flask
+from models import storage
+from api.v1.views import app_views
+import os
+
+
+app = Flask(__name__)
+app.register_blueprint(app_views, url_prefix='/api/v1')
```

Comment:
Perhaps we don't need to add the url prefix here as well?

Reply:
You are right. I removed it